cypress/support/stackable/util.js

```diff
@@ -85,22 +92,38 @@ export function changeControlViewport( viewport = 'Desktop', name = '', isInPopo
 	const selector = () => cy.getBaseControl( name, { isInPopover } )
 	selector()
 		.then( $baseControl => {
-			if ( $baseControl.find( 'button[aria-label="Desktop"]' ).length ) {
-				const hovered = $baseControl.find( 'button[aria-label="Tablet"]' ).length
-				const hover = () => selector().find( 'button[aria-label="Desktop"]' ).trigger( 'mouseover', { force: true } )
-				const selectViewport = () => selector().find( `button[aria-label="${ viewport }"]` ).click( { force: true } )
-				const isActive = () => $baseControl.find( `button.is-active[aria-label="${ viewport }"]` ).length
-				if ( viewport !== 'Desktop' ) {
-					if ( ! hovered ) {
-						hover()
-						selectViewport()
-					} else if ( ! isActive() ) {
+			if ( Cypress.env( 'STACKABLE_VERSION' ) === 2 ) {
+				if ( $baseControl.find( 'button[aria-label="Desktop"]' ).length ) {
+					const hovered = $baseControl.find( 'button[aria-label="Tablet"]' ).length
+					const hover = () => selector().find( 'button[aria-label="Desktop"]' ).trigger( 'mouseover', { force: true } )
+					const selectViewport = () => selector().find( `button[aria-label="${ viewport }"]` ).click( { force: true } )
+					const isActive = () => $baseControl.find( `button.is-active[aria-label="${ viewport }"]` ).length
+					if ( viewport !== 'Desktop' ) {
+						if ( ! hovered ) {
+							hover()
+							selectViewport()
+						} else if ( ! isActive() ) {
+							selectViewport()
+						}
+					} else if ( hovered ) {
 						selectViewport()
 					}
-				} else if ( hovered ) {
-					selectViewport()
+					cy.wait( 1 )
+				}
+			} else if ( Cypress.env( 'STACKABLE_VERSION' ) === 3 ) {
+				if ( $baseControl.find( '.stk-control-responsive-toggle button.is-active' ).length ) {
+					selector().find( '.stk-control-responsive-toggle button.is-active' ).click( { force: true } )
+					selector()
+						.find( '.stk-control-responsive-toggle' )
+						.invoke( 'attr', 'aria-expanded' )
+						.then( ariaExpanded => {
+							if ( ariaExpanded === 'true' ) {
+								selector()
+									.find( `button[aria-label="${ viewport }"]` )
+									.click( { force: true, log: false } )
+							}
+						} )
```

Comment:
Don't use `else if` here.

Prematurely end the function after the first `if` statement instead. With this, the command will still run even if `STACKABLE_VERSION` is not defined.